packages/website/src/pages/form/ColorPicker.tsx

```diff
@@ -1,61 +1,67 @@
+        return (
+            <ColorPicker id="color-picker-id-2" styles={{default: {controls: {display: 'none'}}}} />            
+        );
+    }
+`;
 
-// start-print
+const selector = `
+    import * as React from 'react';
+    import {useSelector} from 'react-redux';
+    import {ColorPicker, PlasmaState, InputSelectors} from '@coveord/plasma-react';
+    
+    export default () => {
+        const selected = useSelector((state: PlasmaState) =>
+            InputSelectors.getValue(state, {
+                id: 'color-picker-id-3',
+            })
+        );
+        return (
+            <>
+                <ColorPicker id="color-picker-id-3" />
+                Selected value: {selected}
+            </>
+        );
+    };
+`;
 
-export const ColorPickerExamples = () => (
-    <PlasmaComponent
+const ColorPickerExamples = () => (
+    <PageLayout
         id="ColorPicker"
         title="Color Picker"
-        usage="A color picker is a visual interface that allows users to select a color."
-        withSource
-    >
-        <div className="mt2">
-            <h2 className="mb2">
-                Built using <a href="https://github.com/casesandberg/react-color/">React Color</a>
-            </h2>
-            <div className="form-group">
-                <label className="form-control-label">Basic ColorPicker</label>
-                <div className="flex">
-                    <ColorPicker id="color-picker-example-1" defaultColor="#F37231" onChangeComplete={logColorPicked} />
-                </div>
-            </div>
-            <div className="form-group">
-                <label className="form-control-label">ColorPicker with different picker styling, hiding controls</label>
-                <div className="flex">
-                    <ColorPicker id="color-picker-example-2" styles={{default: {controls: {display: 'none'}}}} />
-                </div>
-            </div>
-
-            <div className="form-group">
-                <label className="form-control-label">
-                    ColorPickerConnected with color as hex available from state
-                </label>
-                <div className="flex">
-                    <ColorPicker id="color-picker-example-3" defaultColor="#47FF21" />
-                    <Button
-                        className="btn mod-primary ml2"
-                        name="Click me to get color from state"
-                        onClick={() =>
-                            alert(
-                                InputSelectors.getValue(Store.getState(), {
-                                    id: 'color-picker-example-3',
-                                })
-                            )
-                        }
-                    />
-                </div>
-            </div>
-        </div>
-    </PlasmaComponent>
+        section="Form"
+        description={
+            <>
+                A color picker is a visual interface that allows users to select a color. Built using{' '}
```

Comment:
is the `{' '}` useful?

```suggestion
                A color picker is a visual interface that allows users to select a color. Built using
```
```suggestion
                A color picker is a visual interface that allows users to select a color. Built using{' '}
```